models.py

```diff
@@ -0,0 +1,66 @@
+"SQLAlchemhy Models for Can Cook App"
+from flask_bcrypt import Bcrypt 
+from flask_sqlalchemy import SQLAlchemy 
+
+bcrypt = Bcrypt()  
+db = SQLAlchemy()  
+
+def connect_db(app):
+    """
+        Connect this database to provided Flask app.
+        You should call this in your Flask app.
+    """
+    db.app = app
+    db.init_app(app)
+    db.create_all()
+
+# TODO: USER model
+class User(db.Model):
+    """ User Table """
+    __tablename__ = 'users'
+
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.Text, nullable=False, unique=True)
+    email = db.Column(db.Text, nullable=False, unique=True)
+    image = db.Column(db.Text, default = "/asset/default-pic.png")
+    password = db.Column(db.Text, nullable=False)
+
+    # Relationship between User and Favorite
+    favorite = db.Relationship('Favorite', backref="user", cascade="all,delete")
+
+    def __repr__(self):
+        return f"<User #{self.id}: {self.username}, {self.email}>"
+
+    @classmethod
+    def signup(cls, username, email, password, image):
+        """ Sign up user.
+            Hashed password and add user to the system """
+        
+        hashed_pwd = bcrypt.generate_password_hash(password).decode('UTF-8')
+        user = User(username=username, email=email, password=password, image=image)
+        db.session.add(user)
+        return user
```

Comment:
Hello @alexkayabula, currently at this point, this @classmethod doesn’t relate at all to **setting-up databases and tables**. However, I believe in the future, I will add it so that I decided to add it at the beginning. If it isn’t right, I will remove it and add it later when I work on the sign-up/register feature.

Reply:
Ok then, please refactor accordingly and change the label.